### DISCUSSION TITLE
Benefits from curl easy handle reusing

### DISCUSSION THREAD

**Discussion by pkropachev** [curl/curl#13690] [Q&A] [answered]
Hello Team!

Does somebody have experience of reusing curl easy handle (avoid extra calling `curl_easy_init()`) in applications? If so, could you please share information about possible benefits (memory allocation, CPU usage, transfer time reduction and etc).

Thanks!

**Comment by bagder** [comment, marked as answer]
We strongly advise you to reuse the easy handles as much as possible when you use the easy interface (and the multi handle when you use the multi interface). That allows curl to execute faster, use its caches better etc.

**Reply by pkropachev** [reply]
Yes, multi interface is used, but currently easy handle is created for each request before add to multi.
